aqua/qt5/Portfile

```diff
@@ -1920,6 +1920,15 @@ foreach {module module_info} [array get modules] {
             if { ${module} eq "qtsvg" } {
                 patchfiles-append CVE-2023-32573-qtsvg-5.15.diff
             }
+
+            # special case
+            if { ${module} eq "qtspeech" } {
+                # see https://trac.macports.org/ticket/67593
+                # error: library not found for -lasound
+                configure.post_args-append  \
+                    --                      \
```

Comment:
Would it be desirable to also specify `-flite` here, so that the build can only proceed if it successfully configured with flite support enabled?

Reply:
that is implied by the addition of the dependency. I don't see the need to explicitly add default configure arguments but if @MarcusCalhoun-Lopez prefers that, sure that could be done.